server/testdriver/core/src/main/java/org/infinispan/server/test/core/Containers.java

```diff
@@ -31,12 +31,10 @@ public static InetAddress getDockerBridgeAddress() {
 
    public static String imageArchitecture() {
       switch (System.getProperty("os.arch")) {
-         case "amd64":
```

Comment:
I prefer adding a 
```java
case "x86_64":
```
instead of defaulting.

Reply:
done